sql-plugin/src/main/scala/org/apache/spark/sql/rapids/ExecutionPlanCaptureCallback.scala

```diff
@@ -20,10 +20,11 @@ import scala.collection.convert.ImplicitConversions.`collection AsScalaIterable`
 import scala.collection.mutable.{ArrayBuffer, Map => MutableMap}
 import scala.util.matching.Regex
 
-import com.nvidia.spark.rapids.{PlanShims, PlanUtils}
+import com.nvidia.spark.rapids.{GpuAlias, PlanShims, PlanUtils}
+import com.nvidia.spark.rapids.shims.AnsiUtil
```

Comment:
We can't reference a class residing in a parallel world from a class that is loaded by our pyspark code. 

We either have to make sure that 
- either our pyspark/test code loads `ExecutionPlanCaptureCallback` via ShimLoader (don't remember if we had a valid reason to not do it) 
- or  `ExecutionPlanCaptureCallback` uses `ShimLoader` to load AnsiUtils

Reply:
I would guess that the second option is probably cleaner (easier) since it would be a pure Scala approach.